packages/components/src/constants/constants.ts

```diff
@@ -59,3 +60,27 @@ export const FEEDBACK_LINKS = {
   en: 'https://www.hel.fi/helsinki/en/administration/participate/feedback',
   sv: 'https://www.hel.fi/helsinki/sv/stad-och-forvaltning/delta/feedback',
 };
+
+export const DEFAULT_HEADER_MENU_NAME: Record<AppLanguage, string> = {
+  fi:
+    process.env.NEXT_PUBLIC_CMS_HEADER_MENU_NAME_FI ??
+    'Events Helsinki Header FI',
+  en:
+    process.env.NEXT_PUBLIC_CMS_HEADER_MENU_NAME_EN ??
+    'Events Helsinki Header EN',
+  sv:
+    process.env.NEXT_PUBLIC_CMS_HEADER_MENU_NAME_SV ??
+    'Events Helsinki Header SV',
```

Comment:
Events Helsinki Header is the prefix of the name of the header in Events app. If this is now generic, also the default name should be something more generic, like "Header FI", "Header SV", etc. There is currently also an option in the CMS to search a menu with "ID"-parameter, but the ID-value is something that Geniem defines per instance, so we cannot be sure whether it's the same between the multiple app CMS-instances. The name is something that we can self design and set, so it's better to use that one.